esprima.js

```diff
@@ -205,7 +214,12 @@
         DuplicateProtoProperty: 'Duplicate __proto__ fields are not allowed in object literals',
         ConstructorSpecialMethod: 'Class constructor may not be an accessor',
         DuplicateConstructor: 'A class may only have one constructor',
-        StaticPrototype: 'Classes may not have static property named prototype'
+        StaticPrototype: 'Classes may not have static property named prototype',
+        MissingFromClause: 'Missing from clause',
```

Comment:
We don't have import/export declarations in V8 yet, nor in any other engine, we can update those messages in the future.